repository_service_tuf_worker/repository.py

```diff
@@ -251,7 +263,11 @@ def _bump_version(self, role: Metadata) -> None:
         """Bumps metadata version by 1."""
         role.signed.version += 1
 
-    def _update_timestamp(self, snapshot_version: int) -> Metadata[Timestamp]:
+    def _update_timestamp(
+        self,
+        snapshot_version: int,
+        db_targets: Optional[List[targets_models.RSTUFTargets]] = None,
```

Comment:
Can you add a docstring about `db_targets` as the name doesn't show what are you supposed to pass there.